go/pserver/cclient/cclient.go

```diff
@@ -198,44 +194,58 @@ func paddle_get_params(client C.client, names **C.char, dst **C.paddle_parameter
 		return -1
 	}
 
-	for i := 0; i < int(total); i++ {
-		if i >= len(ps) {
-			break
+	names := func() (string, string) {
```

Comment:
It seems that golang's strings#join can do the same thing?
https://golang.org/pkg/strings/#Join